src/core/Akka/Actor/ActorCell.cs

```diff
@@ -544,10 +550,6 @@ private object SerializeAndDeserializePayload(object obj)
                 var manifest = Serialization.Serialization.ManifestFor(serializer, obj);
                 return _systemImpl.Serialization.Deserialize(bytes, serializer.Identifier, manifest);
             }
-            catch (Exception e)
```

Comment:
Move the exception handling to the caller function so we can also capture the envelope